management/persistence.md

```diff
@@ -0,0 +1,330 @@
+  so we are safe.
+  
+* When the child is done rewriting the base file, the parent gets a signal,
+and uses the newly opened increment file and child generated base file to build a temp manifest,
+and persist it.
+
+* Profit! Now FalkorDB does an atomic exchange of the manifest files so that the result of this AOF rewrite takes effect. FalkorDB also cleans up the old base file and any unused increment files.
+
+### How I can switch to AOF, if I'm currently using dump.rdb snapshots?
+
+If you want to enable AOF in a server that is currently using RDB snapshots, you need to convert the data by enabling AOF via CONFIG command on the live server first.
+
+**IMPORTANT:** not following this procedure (e.g. just changing the config and restarting the server) can result in data loss!
+
+Preparations:
+
+* Make a backup of your latest dump.rdb file.
+* Transfer this backup to a safe place.
+
+Switch to AOF on live database:
+
+* Enable AOF: `redis-cli config set appendonly yes`
+* Optionally disable RDB: `redis-cli config set save ""`
+* Make sure writes are appended to the append only file correctly.
+* **IMPORTANT:** Update your `FalkorDB.conf` (potentially through `CONFIG REWRITE`) and ensure that it matches the configuration above.
+  If you forget this step, when you restart the server, the configuration changes will be lost and the server will start again with the old configuration, resulting in a loss of your data.
+
+Next time you restart the server:
+
+* Before restarting the server, wait for AOF rewrite to finish persisting the data.
+  You can do that by watching `INFO persistence`, waiting for `aof_rewrite_in_progress` and `aof_rewrite_scheduled` to be `0`, and validating that `aof_last_bgrewrite_status` is `ok`.
+* After restarting the server, check that your database contains the same number of keys it contained previously.
+
+## Interactions between AOF and RDB persistence
+
+FalkorDB makes sure to avoid triggering an AOF rewrite when an RDB
+snapshotting operation is already in progress, or allowing a `BGSAVE` while the
+AOF rewrite is in progress. This prevents two FalkorDB background processes
+from doing heavy disk I/O at the same time.
+
+When snapshotting is in progress and the user explicitly requests a log
+rewrite operation using `BGREWRITEAOF` the server will reply with an OK
+status code telling the user the operation is scheduled, and the rewrite
+will start once the snapshotting is completed.
+
+In the case both AOF and RDB persistence are enabled and FalkorDB restarts the
+AOF file will be used to reconstruct the original dataset since it is
+guaranteed to be the most complete.
+
+## Backing up FalkorDB data
+
+Before starting this section, make sure to read the following sentence: **Make Sure to Backup Your Database**. Disks break, instances in the cloud disappear, and so forth: no backups means huge risk of data disappearing into /dev/null.
```

Comment:
Please rephrase: "Before starting this section, make sure to read the following sentence: **Make Sure to Backup Your Database**."